### ISSUE TITLE
JSON logging

### ISSUE THREAD

**Issue by actaeon** [newrelic/sidecar#5]
@relistan 

What would you think of an option to enable JSON logging output? I'm interested in writing a Heka decoder to parse Sidecar output, and having an option to have Sidecar logging in JSON would make this a snap.

**Comment by actaeon** [comment]
```
// Significantly modified by Karl Matthias (karl.matthias@gonitro.com) and Rob
// Miller (rmiller@mozilla.com)
```

I see you've been playing with Heka also. I'd love to chat about what you're doing with it.

**Comment by relistan** [comment]
@actaeon Sure, yeah, I've been writing Heka code for the last few months. We can chat about that. Sidecar uses logrus so it seems pretty easy to have a config option to set it to do JSON output.
```log.SetFormatter(&log.JSONFormatter{})```

**Comment by relistan** [comment]
Fixed in 6d0741d37b90265d8449c127e0d75c0613028cde
